packages/reactor-browser/tsconfig.json

```diff
@@ -1,19 +1,27 @@
 {
-  "extends": "../../tsconfig.options.json",
   "compilerOptions": {
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "allowJs": true,
+    "resolveJsonModule": true,
+    "moduleDetection": "force",
+    "isolatedModules": true,
+    "strict": true,
+    "noUncheckedIndexedAccess": true,
+    "outDir": "dist",
+    "sourceMap": true,
+    "declaration": true,
+    "moduleResolution": "Bundler",
+    "target": "ESNext",
+    "module": "ESNext",
+    "noEmit": true,
     "lib": ["ES2022", "DOM"],
-    "emitDeclarationOnly": true,
+    "types": ["node", "./types.d.ts"],
     "jsx": "react-jsx",
-    "outDir": "./dist",
-    "types": ["vitest/globals"],
-    "resolveJsonModule": true,
-    "experimentalDecorators": true
+    "experimentalDecorators": true,
+    "paths": {
+      "document-drive": ["../document-drive/src/index.ts"],
+    }
   },
```

Comment:
doing this will make it include node_modules etc. I usually find its easier to maintain explicit includes than implicit includes with excludes, since you can accidentally add stuff that you meant to exclude and forget to update the array.

Reply:
added fix here: [eb85db7](https://github.com/powerhouse-inc/powerhouse/pull/802/commits/eb85db7fda5a7e7f15fe6946cc81a6742fecef41)